animal_crossing.py

```diff
@@ -0,0 +1,70 @@
+import re
+import sqlite3
+
+import config
+
+if config.bot.acnh_db is not None:
+	db = sqlite3.connect(config.bot.acnh_db)
```

Comment:
reminder to us to get the `config.py` and `config.yaml.example` change in before merging

Reply:
I just updated and pushed the example